spec/models/article_spec.rb

```
@@ -348,6 +348,19 @@ def build_and_validate_article(*args)
             expect(article).to be_valid
           end
         end
+
+        context "when main_image is present and body_markdown is blank" do
```

RuboCop (1.80.2)

[convention] Maximum example group nesting exceeded [5/3]. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NestedGroups)

(RSpec/NestedGroups)